projects/Mallard/src/components/layout/slide-card/index.tsx

```diff
@@ -28,43 +23,56 @@ export const SlideCard = ({
     enabled,
     children,
     onDismiss,
+    getPosition,
 }: {
     enabled: boolean
     children: ReactNode
     onDismiss: () => void
+    getPosition: () => Animated.Value
 }) => {
-    const [scrollY] = useState(() => new Animated.Value(1))
+    const [scrollY] = useState(() => new Animated.Value(0))
     let { current: blocked } = useRef(false)
```

Comment:
I never noticed this destructuring (`{ current: blocked }`), unless I'm misunderstanding this won't properly update the ref? I guess this will have always been this way so probably not a massive issue but might be worth fixing now?